src/DiskIO/IpcIo/IpcIoFile.cc

```diff
@@ -525,6 +533,19 @@ IpcIoFile::HandleNotification(const Ipc::TypedMsgHdr &msg)
         HandleResponses("after notification");
 }
 
+/// Handle messages in IPC queue without relying on the queue state being in
```

Comment:
Same doc update as for CollapsedForwarding::HandleNewDataAtStart

Reply:
Same kind of update in commit 68b76da213296061e22b5a343b29466c8fa4ab3d.

Just like in CollapsedForwarding, there is no message discrimination or multiple queues here.

CollapsedForwarding queue and IpcIoFile queue differ, of course, but we should not be saying "IpcIoFile queue" inside IpcIoFile, especially when no other queue is being discussed. By default, everything is "IpcIoFile" inside IpcIoFile.

IpcIoFile::HandleMessagesAtStart() handles the disker (multi)queue. The same top-level code is used for each end of that queue. That is why there is an `if` statement inside IpcIoFile::HandleMessagesAtStart(). There is still only one (multi)queue here, and HandleMessagesAtStart() handles all (incoming) messages in that single queue.